App/Redux/NotificationRedux.js

```diff
@@ -0,0 +1,29 @@
+/* ------------- Types and Action Creators ------------- */
+
+const ADD_NOTIFICATION = 'ADD_NOTIFICATION'
+const CLEAR_NOTIFICATIONS = 'CLEAR_ALL_NOTIFICATIONS'
+
+export const addNotification = (message) => ({
+  type: ADD_NOTIFICATION,
+  notification: {
+    message,
+    start: Date.now()
+  }
+})
+
+export const clearNotifications = () => ({
+  type: CLEAR_NOTIFICATIONS
+})
```

Comment:
Let's discuss this one. Because the only other action creator here required the custom creator function when using Redux Sauce, I lost the advantage of using Redux Sauce since in their custom creators, you use a string again instead of the constant which defeats the purpose of generating the type string (which is to prevent silent failures). I didn't think it would be a good idea to make this portion of the redux actions/reducers part standard and part redux sauce.